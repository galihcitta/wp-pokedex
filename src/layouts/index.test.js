import React from "react";
import Layouts from "./index";

let wrapper;

beforeEach(() => {
    wrapper = shallowWithTheme(
        <Layouts
            noPadding
            style={{
                header: {
                    withBackButton: true,
                },
            }}
            type="circle"
            size="lg"
        />,
    );
});

describe("Layouts Component", () => {
    it("renders and create snapshot <Layouts />", () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    it("renders success <Layouts />", () => {
        expect(wrapper.exists()).toBe(true);
    });
});
