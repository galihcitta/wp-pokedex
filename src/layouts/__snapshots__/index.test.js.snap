// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layouts Component renders and create snapshot <Layouts /> 1`] = `
<styled.div>
  <styled.div
    noPadding={false}
  >
    <Header
      withBackButton={true}
    />
  </styled.div>
</styled.div>
`;
